aws_lambda_builders/utils.py

```diff
@@ -222,3 +225,53 @@ def extract_tarfile(tarfile_path: Union[str, os.PathLike], unpack_dir: Union[str
                 raise tarfile.ExtractError("Attempted Path Traversal in Tar File")
 
         tar.extractall(unpack_dir)
+
+
+def glob_copy(source: Union[str, list], destination: str) -> None:
+    items = source if isinstance(source, list) else [source]
+    dest_path = Path(destination)
+    known_paths = []
+    for item in items:
+        if os.path.isabs(item.replace('\\', '/')):
```

Comment:
Do we need the `item.replace()`? I think `os.path.isabs()` should already handle this, I could be wrong though.

Reply:
I had to have the replace in there to get the "absolute" detection to work properly under Windows.  "\foo" is considered a relative path, but "/foo" is absolute (go figure).  The purposes of this call is to make sure that the requested asset 's path is relative to the project, so either "\foo" or "/foo" should be treated as non-relative.